docs/portal/client-side-integration.md

```diff
@@ -178,7 +178,13 @@ On the Client-Side Integration page, you can perform the following activities re
 
 ### Adding Mobile App IDs
 
-It's important to provide a complete list of all your mobile app IDs, including Android App IDs, iOS/tvOS Bundle IDs, and iOS App Store IDs. UID2 token requests are accepted only from mobile app IDs that are set up in the UID2 Portal.
+It's important to provide a complete list of all your mobile app IDs, including:
+- Android App ID: any that apply.
+- iOS/tvOS Bundle ID and corresponding iOS App Store ID: any that apply.
+
+:::important
+UID2 token requests are accepted only from sources where the applicable IDs are set up in the UID2 Portal.
+:::
```

Comment:
Instead of "set up" here, I would say "added" since that's the title of the section.  I think "set up" makes it sound like a different task

Reply:
Done, thx!